src/strategies/richquack-staked-amount/index.ts

```diff
@@ -0,0 +1,57 @@
+import { BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller } from '../../utils';
+
+export const author = 'ephdtrg';
+export const version = '0.1.0';
+
+const abi = [
+  {
+    inputs: [
+      {
+        internalType: 'address',
+        name: '',
+        type: 'address'
+      }
+    ],
+    name: 'stakeInfo',
+    outputs: [
+      {
+        internalType: 'uint256',
+        name: 'level',
+        type: 'uint256'
+      },
+      {
+        internalType: 'uint256',
+        name: 'totalStakedForUser',
+        type: 'uint256'
+      }
+    ],
+    stateMutability: 'view',
+    type: 'function'
+  }
+];
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+): Promise<Record<string, number>> {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+  const multi = new Multicaller(network, provider, abi, { blockTag });
+  addresses.forEach((address) =>
+    multi.call(address, options.stakingAddress, 'stakeInfo', [address])
+  );
+  const result: Record<string, BigNumberish> = await multi.execute();
+
+  return Object.fromEntries(
+    Object.entries(result).map(([address]) => [
+      address,
+      parseFloat(formatUnits(result[address][0], 18))
+    ])
```

Comment:
Hi @ephdtrg i think contract-call strategy should work for you, can try?

Reply:
Hi, @ChaituVR ! Thank you for response

Do you mean to use existing contract-call strategy for this workflow, instead of writing new strategy, correct?

Reply:
Yep @ephdtrg Can use it in your space directly :)

Reply:
Thank you for suggestion @ChaituVR !
Was not aware of this strategy, will check it out and close this PR if it works as intended  :)